windows/src/desktop/setup/UfrmRunDesktop.dfm

```diff
@@ -445,84 +445,116 @@ object frmRunDesktop: TfrmRunDesktop
     Align = alClient
     BevelOuter = bvNone
     TabOrder = 0
-    DesignSize = (
-      699
-      208)
-    object lblActions: TLabel
-      Left = 20
-      Top = 189
-      Width = 149
-      Height = 13
-      Anchors = [akLeft, akBottom]
-      Caption = 'Setup will download and install:'
-    end
-    object panAction: TPanel
+    object panActionText: TPanel
       Left = 0
-      Top = 89
-      Width = 699
-      Height = 57
-      Align = alTop
+      Top = 0
+      Width = 272
+      Height = 208
+      Align = alClient
       BevelOuter = bvNone
       TabOrder = 0
-      object cmdInstall: TButton
-        Left = 200
-        Top = 14
-        Width = 298
-        Height = 40
-        Caption = 'ssInstallButton'
-        Font.Charset = DEFAULT_CHARSET
-        Font.Color = 2640557
-        Font.Height = -18
-        Font.Name = 'Tahoma'
-        Font.Style = [fsBold]
-        ParentFont = False
+      ExplicitWidth = 347
+      DesignSize = (
+        272
+        208)
+      object sbActionText: TScrollBox
+        Left = 20
+        Top = 6
+        Width = 246
+        Height = 196
+        HorzScrollBar.Visible = False
+        Anchors = [akLeft, akTop, akRight, akBottom]
+        BorderStyle = bsNone
+        ParentBackground = True
         TabOrder = 0
-        OnClick = cmdInstallClick
+        ExplicitWidth = 321
+        object lblActions: TLabel
+          Left = 0
+          Top = 18
+          Width = 149
+          Height = 13
+          Caption = 'Setup will download and install:'
```

Comment:
Should this be something like `ssDownloadInstallCaption`?

Reply:
It could be, but it's not important because it is overwritten at runtime anyway.